### PR TITLE
Add C# Hello World

### PR DESCRIPTION
**Pull request by JipSterk**
#1

### CODE REVIEW

File: C#.cs [hunk 1]
```diff
@@ -0,0 +1,9 @@
+using System;
+
+public class Main
+{
+    static void Main()
+    {
+        Console.WriteLine("Hello World!");
```
Comment (TiagoDanin) [inline C#.cs]: Does not display `Hello World`, see [contributing guide](https://github.com/knightking100/hello-worlds/blob/master/CONTRIBUTING.MD)

File: C#.cs [hunk 2]
```diff
@@ -0,0 +1,9 @@
+using System;
+
+public class Main
+{
+    static void Main()
```
Comment (TiagoDanin) [inline C#.cs]: Error: Member names cannot be the same

File: C#.cs (hunk 1 above)
Comment (knightking100) [reply C#.cs]: That's also ok

**Comment by knightking100** [comment]
Come on, no perfect c# hello world? I will do it myself

**Pull request merged**